caffe2/onnx/onnx_exporter.cc

```diff
@@ -364,6 +364,67 @@ std::vector<NodeProto> OnnxExporter::CreateGemmNode(
 
   return nodes;
 }
+
+void OnnxExporter::InitOpToTensorProto(
```

Comment:
Ok. I'm not aware of the predictor exporter and it's workflow. I did this because our c2 model zoo came in as init/pred net pair and I was biased on that. I agree it's more generic. I'll collect more info on this. Change itself shouldn't be too difficult.